tests/test_events_ops.py

```diff
@@ -0,0 +1,174 @@
+import unittest
+import json
+from app import createApp, db
+from . import test_app_tests
+import base64
+
+class EventsActivitiesTestCases(unittest.TestCase):
+
+    def setUp(self):
+        self.app = createApp(conf_name='testing')
+        self.client = self.app.test_client
+
+        self.event1 = {
+            "name" : "Partymad",
+            "location" : "Nairobu",
+            "description" : "here and 2",
+            "date": "2017-12-24",
+            "cost" : 2000,
+            "category": "Indoors"
+        }
+        self.event2 = {            
+            "name" : "Partymad",            
+            "description" : "here and 2",
+            "date": "2018-01-05",
+            "cost" : 2000
+        }
+        self.event3 = {                        
+            "location" : "Nairobu",
+            "description" : "here and 2",
+            "date": "10/10/2017",
+            "cost" : 2000
+        }
+        self.event4 = {            
+            "name" : "Partymad",
+            "location" : "Nairobu",
+            "description" : "here and 2",
+            "date": "2017-10-10",
+            "cost" : 2000
+        }
+        self.rsvp_ = {            
+            "eventid":1,            
+            "rsvp":"attending"
+        }
+
+        self.user = {
+            'email': 'test@kungu.com',
+            'username':'test',
+            'password':'hardpass'
+        }
+        self.login_details = {
+            'username' : 'test',
+            'password' : 'hardpass'
+        }
+
+        """Unit tests for events goes here"""    
+
+     # binds the app to the current context
+        with self.app.app_context():
+            # create all tables
+            db.create_all()
+
+    #register a user
+    def register_users(self):
```

Comment:
Does this test work, it is not asserting anything

Reply:
This isn't a test. It's a function to register a user who will be logged in by another function to get the token. That token is being used by the events endpoints tests.